libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFCryptChunks.h

```diff
@@ -0,0 +1,94 @@
+//
+//  SFCryptChunks.h
```

Comment:
Could you switch the copyright to the standard ```Mobile SDK``` copyright? You should be able to grab it from one of the other files.